frontend_lib/src/mention.js

```diff
@@ -95,7 +95,7 @@ export const addClassToMentionsOfUser = (rawContent, username, userClassName = M
 
   const releventMentions = [username, ...GROUP_MENTION_TRANSLATION_LIST]
```

Comment:
s/relevent/relevant/:

```suggestion
  const relevantMentions = [username, ...GROUP_MENTION_TRANSLATION_LIST]
```

Reply:
One day, I'll know how to write relevant. ^^

Reply:
fixed